packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -263,6 +263,31 @@ void main() {
         expect(scrollView.controller, primary2);
       });
 
+      testWidgets('Use custom ScrollController when scrollController is set', (WidgetTester tester) async {
+        final ScrollController customController = ScrollController(initialScrollOffset: 10);
+        await tester.pumpWidget(
+          MaterialApp(
+            home: ReorderableListView(
+              scrollController: customController,
+              children: const <Widget>[
+                SizedBox(width: 100.0, height: 100.0, child: Text('C'), key: Key('C')),
+                SizedBox(width: 100.0, height: 100.0, child: Text('B'), key: Key('B')),
+                SizedBox(width: 100.0, height: 100.0, child: Text('A'), key: Key('A')),
+              ],
+              onReorder: (int oldIndex, int newIndex) { },
+            ),
+          )
+        );
+
+        final ReorderableListView listView = tester.widget(
+          find.byType(ReorderableListView),
+        );
+
+        listView.scrollController.jumpTo(20);
+
+        expect(customController.offset, 20);
+      });
+
```

Comment:
You will also want to test dragging the list view and seeing if the controller's offset values update with changes on the screen. I did this one since it's a little odd. The first 20px of dragging seems to be ignored by the `ReorderableListView`:
```dart
        // Drag the UI to see if the scroll controller updates accordingly
        await tester.drag(
          find.text('B'),
          const Offset(0.0, -100.0),
        );
        listViewTopLeft = tester.getTopLeft(
          find.byType(ReorderableListView),
        );
        firstBoxTopLeft = tester.getTopLeft(
          find.byKey(firstBox),
        );
        // Initial scroll controller offset: 40.0
        // Drag UI by 100.0 upwards vertically
        // First 20.0 px always ignored, so scroll offset is only
        // shifted by 80.0.
        // Final offset: 40.0 + 80.0 = 120.0
        expect(customController.offset, 120.0);
```

Reply:
You've taught me A LOT about the capabilities of flutter's testing library. Man lol, I need to go back over the docs a bit because a lot of this is new to me.